arkane/mainTest.py

```diff
@@ -79,6 +80,31 @@ def test_arkane_examples(self):
             error_message += '{1} in {0}; '.format(example_name, example_type)
         self.assertFalse(self.failed, error_message)
 
+    def test_arkane_two_parameter_arrhenius_fit(self):
+        test_path = os.path.join(self.test_base_path, 'two_parameter_arrhenius_fit')
+        file_to_remove = ['output.py', 'chem.inp', 'supporting_information.csv']
```

Comment:
this clean-up should occur under `tearDownClass` to leave the repo clean after running the tests
You could also leave it here in addition to `tearDownClass` if these files aren't being overwritten

Reply:
These files are not being overwritten. I will leave this cleanup here.

Reply:
OK to leave it here, but ALSO implement it under `tearDownClass` to leave the repo clean